public/javascripts/controller/report.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Navigation related controller and functionality
+ */
+
+angular.module('undp-ghg-v2')
+  .controller('ReportController', ['$scope', '$location', '$routeParams','ReportFactory', '$sce', '$http', '$cookies',
+    function($scope, $location, $routeParams, ReportFactory, $sce, $http, $cookies) {
+
+      //model
+      angular.extend($scope, {
+        'reports': [],
+        'currentReportUrl': ''
```

Comment:
this URL should be passed from the backend - via some environment configuration....